### PR TITLE
A Year in the Life of a Holiday Extras Product Owner

### PR DESCRIPTION
**Pull request by jodiedoubleday**
### Blog - "A Year in the Life of a Holiday Extras Product Owner"
Wrote this little look-back on 2014 as a Product Owner and my recommended list of events.
Would love someone to review the post for me:)
*PR on behalf of Anna Divers*
#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by andrew-c** [comment]
:+1: Might be worth putting in links to Twitter handles.

File: _posts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+**November 2014**- a hop over to Dublin for <http://websummit.net/> - much like Appsworld, a massive event needing several of you to divide and conquer.  Very much a look into the future and what’s coming next in most instances.  Two favourite talks - firstly _From Monkey Coders to Smart Gorillas_ courtesy of Traity - a framework for always keeping the big picture in sight, they presented a neat way to communicate the whole picture from top line vision to functionality and how it fits together, and gave a warning of the dangers of getting lost in small tasks and losing sight of the big picture.  The other was from the founder of <https://trueandco.com/> all about _building trust and friendly data collection_ - and asked us what is the reward for the customer by giving us extra data?  What is our double handshake, how do we build a virtuous cycle of data?
+
+**December 2014** - hooked up with an old friend and associate Gwyn Morfey (formerly MD of New Bamboo) to talk about how best to scale and manage Scrum teams, was reassuring to talk through our transformation in the web development space this year and hear we were doing a lot of things right!
+
+_So what’s next in 2016?_
```
Comment (jackdcrawford) [inline _posts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md]: Should this be 2015?

File: _posts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+categories: productowner
+author: Anna Divers
+tags: Product-Owner Lean Agile Product-Development
+time: 8 mins
+comments: true
+---
+
+## A Year in the Life of a Holiday Extras Product Owner and my recommended events for all you PO's out there!
+
+**2014** - **a year of travelling, learning and transforming**.  My New Years Resolution last year was to get out of the building more, learn more from external sources after the previous year very much head down on a conversion and traffic-migration mission.  Here’s how it turned out...
```
Comment (alannamooney) [inline _posts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md]: Years should be Year's

**Comment by alannamooney** [comment]
Taking second editor review

File: _posts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+**February 2014** - Holiday Extras’s ‘mobile moment’ has not happened just yet but with it around the corner, we took a trip to Appsworld in San Francisco <https://www.apps-world.net/northamerica/>.  This event demonstrated just how fast moving and diverse the mobile app space is and just how much there is to learn, or to quote [@lukew](http://www.twitter.com/lukew) ‘unlearn’.  A huge event, too much to cover by just the three of us who went - later in the year some of our developers went to the London event, so we’ve covered most of the content.  The highlight has to have been listening to Steve Wozniak being interviewed - two points really resonated as a Product Owner - design is important - how something looks is very important - looks can convince people to buy it, and this applies to both the company and the product.  And his other point is that is real important to know what features not to include - the ability to say No is critical.
```
Comment (alannamooney) [inline _posts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md]: Should "and his other point is that is real important" be "and his other point is that it's real important"?

File: _posts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+ps.  San Francisco is an amazing place, I fell in love with it.  If you get a chance, go!
+
+
+**March 2014** - Found a neat (new-to-me) lean product development conference - LeanDayLondon https://leanday.splashthat.com/ (found through the equally resourceful site for product owners <http://www.mindtheproduct.com/>).  Really recommend this 2-day conference - high quality talks and I loved the fact that the content was also aimed at applying lean in large, established companies and not merely start-ups alone.  Loved listening to Dr Tendayi Viki talk about the Ambidextrous Organisation and the biggest challenge for large organisations trying to innovate is their size - they have to keep their cash cow going but innovate at the same time (they need to develop new cows).
```
Comment (alannamooney) [inline _posts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md]: Should "Ambidextrous Organisation and the biggest challenge for large organisations" be "Ambidextrous Organisation and how the biggest challenge for large organisations"

File: _posts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+ps.  If you ever get to meet Jeff ask him about the Leafblower incident.
+
+
+**October 2014** - second highlight of the year was in the form of Luke Wroblewski, who spent 2 days with us, educating us on the mobile landscape and how to design for multi-device.  Never known someone to know their subject matter so deeply and broadly as Luke - no kidding he came with a 451-slide deck!  Great insight for the 30 of us who had the privilege of participating his workshop - can see why Google snapped him and his start-up up!
```
Comment (alannamooney) [inline _posts/2015-01-12-a-year-in-the-life-of-a-holiday-extras-product-owner.md]: "Great insight for the 30 of us who had the privilege of participating his workshop" should be "Great insight for the 30 of us who had the privilege of participating in his workshop"

**Pull request merged**